ui/app/components/sidebar/nav/tools.hbs

```diff
@@ -19,4 +19,10 @@
       />
     {{/if}}
   {{/each}}
+  <Nav.Link
+    @route="vault.cluster.tools.open-api-explorer.index"
+    @current-when="vault.cluster.tools.open-api-explorer.index"
```

Comment:
I pulled down the branch to test and you can remove `.index` from the route and also `current-when` is not necessary since the link only needs to be active when the route is navigated to.

Reply:
Thanks for checking. I'll update.